### ISSUE TITLE
Wrong cache if customer is in several customer groups

### ISSUE THREAD

**Issue by wucherpfennig** [litespeedtech/lscache_prestashop#39]
Dear all

Thank you for this amazing module. We are using it and have been happy with it till recently.

We noticed that there are issues with wrong caches delivered if you work with several customer groups for discounting purposes etc.

example: 
User JohnDoe is in group A + C
User JaneDoe is in group B + C

we noticed that it can happen that now Jane Doe (gr. B) might see discount from JohnDoe (gr. A) since they got served by the same customer group 'C'.

We did not have this issue with other caching modules which provide an additional option to generate cache according to all selected customer groups.

I hope I was able to explain myself.

BR wucherpfennig

**Comment by wucherpfennig** [comment]
Hi everyone

Is this repository abandoned?

BR wucherpfennig

**Comment by litespeedtech** [comment]
currently it uses $context->customer->id_default_group; I think it still won't work for your case, right? and your code snippet looks fine for me. Let me know if you are still using the plugin and we will look into it. 
Very sorry for late reply.

**Comment by wucherpfennig** [comment]
Yes and no. I think this is still a valid and a fixable request but no we currently use another module (although it does not work as well).
